deps/wazuh_testing/wazuh_testing/remote.py

```diff
@@ -2,15 +2,26 @@
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 
+import os
 import socket
 
+import wazuh_testing.tools.agent_simulator as ag
+
 from wazuh_testing.tools import ARCHIVES_LOG_FILE_PATH
-from wazuh_testing.tools.file import truncate_file
-from wazuh_testing.tools.monitoring import FileMonitor, make_callback, REMOTED_DETECTOR_PREFIX
+from wazuh_testing.tools import file
+from wazuh_testing.tools import monitoring
+from wazuh_testing.tools.services import control_service
+
 
 UDP = "UDP"
 TCP = "TCP"
 TCP_UDP = "TCP,UDP"
+REMOTED_GLOBAL_TIMEOUT = 10
+EXAMPLE_MESSAGE_EVENT = '1:/root/test.log:Feb 23 17:18:20 35-u20-manager4 sshd[40657]: Accepted publickey for root' \
+                        ' from 192.168.0.5 port 48044 ssh2: RSA SHA256:IZT11YXRZoZfuGlj/K/t3tT8OdolV58hcCOJFZLIW2Y'
+EXAMPLE_MESSAGE_PATTERN = 'Accepted publickey for root from 192.168.0.5 port 48044'
+SOCKETS_PATH = '/var/ossec/queue/sockets'
```

Comment:
This should be defined in the `wazuh_testing.tools.__init__.py` with the rest of the other variables.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1089/commits/d308661a2d36e156dd6fc70ff8363b0e3ef23527